### DISCUSSION TITLE
Youtube does not receive RTMP signal

### DISCUSSION THREAD

**Discussion by eas3011** [ant-media/Ant-Media-Server#6063] [FAQs]
I am testing Ant-Media Server on an AWS instance, I have configured all the ports and reviewed them several times. I reinstalled it a few times, and everything seems to be perfect, however, YouTube doesn't receive the streaming signal. Is there a minimum configuration for installing on an AWS instance or what could be happening?

**Comment by yashtandon113** [comment]
@eas3011 
Please make sure that your stream has audio, as YouTube does not accept streams without audio. Also, their should be no blank space in the URL while adding the RTMP endpoint.

If everything appears to be in order, please send your server credentials to me at yash.tandon@antmedia.io so I can check on the issue.

**Reply by eas3011** [reply]
I reinstalled the package on the AWS, Google Cloud and Linode instances, uploaded the videos, configured YouTube's RTMP as recommended, created the Plaaylits, but none of them started the stream. I used an instance with Ubuntu 20 on 2CPU and 4GB of memory and 30GB of storage

**Comment by eas3011** [comment]
I installed AMS Community Edition 2.8.1 20240130_0746 on an AWS t2.medium instance (2vCPUs 4RAM Ubuntu), the inbound rules for TCP: 1935, 5554, 5443, 5080; SSH: 22; HTTP: 80; HTTPS: 443. I've reinstalled AMS a few times on new instances on AWS, but none of them have been successful. I’m  looking for a solution. Thank you

**Reply by yashtandon113** [reply]
@eas3011
Are you able to access the server after installing it? If yes, are you trying to create a playlist and stream it to YouTube?

If so, please make sure that your VOD/MP4 file has audio, as YouTube does not accept streams without audio.
